.drone.star

```diff
@@ -70,35 +70,62 @@ config = {
 }
 
 def main(ctx):
+
 	before = beforePipelines()
 
-	stages = stagePipelines()
+	coverageTests = coveragePipelines(ctx)
+	if (coverageTests == False):
+		print('Errors detected in coveragePipelines. Review messages above.')
+		return []
+
+	dependsOn(before, coverageTests)
+
+	stages = stagePipelines(ctx)
 	if (stages == False):
-		print('Errors detected. Review messages above.')
+		print('Errors detected in stagePipelines. Review messages above.')
 		return []
 
 	dependsOn(before, stages)
 
-	after = afterPipelines()
-	dependsOn(stages, after)
+	afterCoverageTests = afterCoveragePipelines(ctx)
+	dependsOn(coverageTests, afterCoverageTests)
 
-	return before + stages + after
+
+	after = afterPipelines(ctx)
+	dependsOn(afterCoverageTests + stages, after)
+
+	return before + coverageTests + afterCoverageTests + stages + after
 
 def beforePipelines():
 	return codestyle() + jscodestyle() + phpstan() + phan()
 
-def stagePipelines():
+def coveragePipelines(ctx):
+	# All pipelines that might have coverage or other test analysis reported
+	jsPipelines = javascript(ctx)
+	phpunitPipelines = phptests(ctx, 'phpunit')
+	phpintegrationPipelines = phptests(ctx, 'phpintegration')
+	if (jsPipelines == False) or (phpunitPipelines == False) or (phpintegrationPipelines == False):
+		return False
+
+	return jsPipelines + phpunitPipelines + phpintegrationPipelines
+
+def stagePipelines(ctx):
 	buildPipelines = build()
-	jsPipelines = javascript()
-	phpunitPipelines = phptests('phpunit')
-	phpintegrationPipelines = phptests('phpintegration')
-	acceptancePipelines = acceptance()
+	jsPipelines = javascript(ctx)
+	phpunitPipelines = phptests(ctx, 'phpunit')
+	phpintegrationPipelines = phptests(ctx,'phpintegration')
```

Comment:
These are now in `coveragePipelines` - we don't need to run them again here.
That might be the reason for drone reporting "linter: duplicate pipeline names"